pkg/TIPA.py

```
@@ -85,15 +86,27 @@ def run(self) -> None:
+            interact_key = settings.get("interact_key", "f")
+
+        keyboard.on_press_key(key=interact_key, callback=self.on_release)
 
         # Take the screenshot for the item name (in inventory/stash)
         while not self.need_quit:
             if not self.listen:
                 self.resumeEvent.wait()
                 self.listen = True
+
+                keyboard.unhook_key(interact_key)
+                with open("_internal/settings.json", "r") as settings_file:
```

Ruff (0.8.0)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)